apps/alerting/notifications/pkg/apis/alerting_manifest.go

```diff
@@ -7,63 +7,56 @@ package apis
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/grafana/grafana-app-sdk/app"
 	"github.com/grafana/grafana-app-sdk/resource"
 
 	v0alpha1 "github.com/grafana/grafana/apps/alerting/notifications/pkg/apis/alerting/v0alpha1"
```

Comment:
@IfSentient the folder is generated at `pkg/apis/alerting/v0alpha1` and the import here is generated with `pkg/apis/notifications/v0alpha1`. i had to manually change the import

Reply:
This should be fixed in the latest `apiserver-custom-routes`